src/cgame/cg_consolecmds.cpp

```diff
@@ -432,6 +432,17 @@ static void CG_BeaconMenu_f()
 	Rocket_DocumentAction( rocketInfo.menu[ ROCKETMENU_BEACONS ].id, "show" );
 }
 
+static void CG_BotCommandMenu_f()
+{
+	if ( !cg.snap
+		|| cg.clientNum != cg.snap->ps.clientNum /* following */
+		|| cg.snap->ps.persistant[ PERS_SPECSTATE ] != SPECTATOR_NOT /* spectating */ )
```

Comment:
Is it supposed to be disabled when you are dead? If so why?

Reply:
> Is it supposed to be disabled when you are dead?

No.